xrdpvr/xrdpvr.c

```diff
@@ -24,6 +24,27 @@
 #include "xrdpvr.h"
 #include "xrdpvr_internal.h"
 
+/* Support newer versions of ffmpeg and libav */
+#if ! defined(CODEC_TYPE_AUDIO)
```

Comment:
xrdp appears to standardize on "!defined" without a space. Please follow the existing style.